com/metacube/shoppingcart/main/ProductView.java

```diff
@@ -0,0 +1,56 @@
+package com.metacube.shoppingcart.main;
+
+import java.util.Scanner;
+
+import com.metacube.shoppingcart.controller.ProductController;
+import com.metacube.shoppingcart.login.LoginAsAdmin;
+import com.metacube.shoppingcart.login.LoginAsUser;
+/**
+ * it is our main class where we have view to consol
+ * @author Vinod Patel
+ *
+ */
+public class ProductView {
+	static int choice=0;
+    static int choice2=1;
+    static Scanner input = new Scanner(System.in);    	
+    // Here we create a object of singleton class ProductController
+	static ProductController productController = ProductController.getInstance();
+    
+	public static void main(String[] arg){
```

Comment:
Code within main method should be in the try catch bolck so that any where in the program if error is thrown will be caught here. You no need to put try catch any where else.